pset-iterators.js

```diff
@@ -87,7 +92,10 @@ Tip: your new array should contain 45 elements
 console.log('Problem 2:')
 
 // Add your code below this line
-
+const presidentialParties = presidents.filter(function(parties) => {
```

Comment:
@jimernest90 the iterator method you want to use here is `.map()` since the problem asked for a full list of the parties of each president. See [this implementation](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-iterators.js#L98) for more details